packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -53,74 +52,47 @@ const NotificationDetail = () => {
     label: string;
     rawValue: string | undefined;
     value: ReactNode;
-  }> = [
```

Comment:
Fare attenzione a quando si fanno le merge. Probabilmente per risolvere dei conflitti, si sono perse delle modifiche provenienti da altre pr.
Ripartire da quello che è presente su develop e riapplicare le modifiche utili a risolvere il bug in questione.
Se può essere utile, scaricare l'estensione Git History diff di visual studio code e confrontare le due versioni del file (develop e branch corrente)

Reply:
Andrea scusa ma non mi pare di vedere regressioni, le specifiche per la visualizzazione sono cambiate (vedi anche figma linkato nella issue). A cosa ti riferisci in particolare?

Reply:
Alle label